src/bin/cargo/commands/clean.rs

```diff
@@ -17,29 +21,243 @@ pub fn cli() -> Command {
         .arg(
             flag(
                 "dry-run",
-                "Display what would be deleted without deleting anything",
+                "Display what would be deleted without deleting anything (unstable)",
             )
```

Comment:
Why was this marked `unstable`?

Reply:
This was written before #12638, where it was decided to go ahead and stabilize the dry-run option. After rebasing, this should be resolved.